src/recipes/thought.ts

```diff
@@ -25,7 +25,6 @@ const thoughtRecipe = defineRecipe({
         /**
           .single-line element has display: inline-block property, it affects the height calculation of the .thought div so we need to set the height manually.
          */
-        height: 'minThoughtHeight',
```

Comment:
Make sure adjust inline comments when you change code. Otherwise they won't make sense anymore.